### ISSUE TITLE
MacOS not working after new update

### ISSUE THREAD

**Issue by Confuzzer** [solsticegamestudios/GModCEFCodecFix#96]
After updating Gmod I tried to open it and was met with this. I've used the script and it worked beforehand but now it just crashes and turns my screen blue. Strangely the blue doesn't appear in my screen recording though. 

https://github.com/solsticegamestudios/GModCEFCodecFix/assets/104530297/f4bf11d6-cae3-4c26-9d89-44f6f7cf394a

**Comment by AkikoKumagara** [comment]
Duplicate of #73 and #82 

This isn't a bug or an issue. You need to wait for the manifest to be updated with new file checksums before GModCEFCodecFix will be able to run successfully again. Once this is done and GModCEFCodecFix is run once again, everything should work. Because of macOS builds of GMod using codesign, this will happen everytime GMod's x86-64 branch is updated, as the files checksums are changed even if the files themselves are not different.

**Comment by AkikoKumagara** [comment]
@Confuzzer run GModCEFCodecFix-macOS again, it should work now. If you still have issues, validate your game files on Steam, wait for them to download, then run the executable one last time.
